### PR TITLE
deleted events

### PR DESCRIPTION
**Pull request by mohitvijay45**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -4,25 +4,111 @@ properties([
...
+    env.BEFTA_S2S_CLIENT_ID = "ccd_data"
+    env.ADOPTION_WEB_CLIENT_ID = "adoption-web"
+    env.ADOPTION_WEB_MICROSERVICE = "adoption_web"
+    env.CCD_API_GATEWAY_OAUTH2_REDIRECT_URL = "https://www-ccd.aat.platform.hmcts.net/oauth2redirect"
+    env.TEST_S2S_URL = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
+}
+
+onMaster {
+    env.ENVIRONMENT="aat"
+    env.CASE_API_URL="http://adoption-cos-api-aat.service.core-compute-aat.internal"
```
[the remark this answers is not in the document]
Comment (ptrelease) [reply Jenkinsfile_nightly:81]: @gokul-sol What do you think?

[the remark this answers is not in the document]
Comment (gokul-sol) [reply Jenkinsfile_nightly:81]: @ptrelease oh oops, missed to change in Jenkinsfile.. that adoption-aat url is redundant now I think, it's not loading any page..

[the remark this answers is not in the document]
Comment (ptrelease) [reply Jenkinsfile_nightly]: Thanks @gokul-sol .  I've checked further and CASE_API_URL is used to [define the callback host](https://github.com/hmcts/adoption-cos-api/blob/fc5c24c7bdaf97105d1878e7b549b70ccb46aa1b/src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/Adoption.java#L36).

So it makes _more_ sense if the url is http://adoption-cos-api-aat.service.core-compute-aat.internal, which works fine still.

**Comment by ptrelease** [comment]
gokul-sol  has reverted the case api url for AAT in his last commit.  This suggests that if AAT passes then Master should not.

**Review by ptrelease — APPROVED** [review]

File: Jenkinsfile_nightly (hunk 1 above)
[the remark this answers is not in the document]
Comment (ptrelease) [reply Jenkinsfile_nightly]: Discussed. @gokul-sol has reverted AAT.  If that works - will revert others.

**Pull request merged**